.github/workflows/build-images-base.yaml

```diff
@@ -52,6 +52,11 @@ jobs:
           persist-credentials: false
           ref: ${{ github.event.pull_request.head.sha }}
 
+      - name: Set-up git
+        run: |
+          git config user.name "GH action"
+          git config user.email "ghabot@does.not.exist.cilium.org"
```

Comment:
Do we want to create an actual bot account for this and not necessary for now?

Reply:
nit: While we're at it, maybe have a more descriptive name like `Cilium Imagebot` and use `noreply@cilium.io` since we don't control cilium.org?